js/demo-app/package.json

```diff
@@ -16,7 +16,18 @@
   },
   "author": "",
   "license": "MIT",
+  "browserslist": [
```

Comment:
This may not be our exact supported list, but it's a decent start.  IE10 might be opening pandora's box, so I left it off this list as a first pass.  Still need to get 11 working correctly anyway ;)